src/renderers/dom/shared/inputValueTracking.js

```diff
@@ -133,9 +133,10 @@ var inputValueTracking = {
       return false;
     }
     var tracker = getTracker(subject);
+    const isNode = (subject: any).nodeType === ELEMENT_NODE;
 
     if (!tracker) {
-      if (typeof (subject: any).tag === 'number') {
+      if (isNode || typeof (subject: any).tag === 'number') {
```

Comment:
Also, isn't this check backward? This logic means that if `subject` is a Fiber _or_ if `subject` is a DOM node, we're passing `subject.stateNode` to `trackNode`.

Won't `subject.stateNode` be `undefined` for DOM nodes?